gnma/applet.py

```diff
@@ -84,6 +84,9 @@ def sort_and_filter_event(self):
                     "I_CAL_STATUS_CONFIRMED"):
                 logging.debug("[SKIP] non confirmed event")
                 continue
+            if (self.config["skip_all_day"] and event.all_day == True):
```

Comment:
i think pylint is may be going to tel us that parenthesis are not necessary.

Reply:
the pylint error is :
```
gnma/applet.py:87:48: C0121: Comparison 'event.all_day == True' should be 'event.all_day is True' if checking for the singleton value True, or 'bool(event.all_day)' if testing for truthiness (singleton-comparison)
```

i think that should do it : 

```suggestion
            if self.config["skip_all_day"] and event.all_day:
```